kasa/tests/test_bulb.py

```diff
@@ -232,3 +232,17 @@ async def test_non_dimmable(dev):
         assert dev.brightness == 0
     with pytest.raises(SmartDeviceException):
         await dev.set_brightness(100)
+
+
+@bulb
+async def test_ignore_default_not_set_without_color_mode_change_turn_on(dev, mocker):
+    query_helper = mocker.patch("kasa.SmartBulb._query_helper")
+    # When turning back ignore do not ignore
+    # default so we restore state
```

Comment:
```suggestion
    # When turning back without settings, ignore default to restore the state
```
Maybe? This should probably be a docstring for the method.

Reply:
Change looks good.  I'm not sure it needs to be in the docstring as I think its implementation details that the caller doesn't need to care about since they don't need to change what they are passing in.